ethers-core/src/types/chain.rs

```diff
@@ -22,6 +22,7 @@ pub enum Chain {
     XDai = 100,
     Polygon = 137,
     Fantom = 250,
+    Ganache = 1337,
```

Comment:
this is also used by by geth and hardhat,
so I think we should make this `Dev` or something
we can't use Geth,Ganache, Hardhat because we have repr(u64)

Reply:
Makes sense, changed to `Dev`